Rakefile

```diff
@@ -3,7 +3,11 @@
 require "bundler/gem_tasks"
 require "rspec/core/rake_task"
 require "standard/rake"
+require "appraisal"
 
 RSpec::Core::RakeTask.new(:spec)
-
 task default: :spec
+
+if !ENV["APPRAISAL_INITIALIZED"]
```

Comment:
**question:** I haven't seen this ENV variable being used. Can you share a bit, why you're adding this? It would be wonderful to have a note in README.md, if this is related to local setup.

Reply:
I've followed the guide that comes from Appraisal. I believe the way the do it is that when you run a Rake task "from the outside" this env var is not set (for example you would do this when doing `bundle exec rake appraisal`. But when Appraisal _itself_ runs a Rake task it does set the envvar so that the Rake task executes whatever is configured in it "normally".

Or, at least, that's how I think it's supposed to work.

Reply:
Here is what I found in appraisal readme.

```md
In the case that you want to run all the appraisals by default when you run
`rake`, you can override your default Rake task by put this into your Rakefile:

    if !ENV["APPRAISAL_INITIALIZED"] && !ENV["TRAVIS"]
      task :default => :appraisal
    end
```

`rake` command will run all appraisals now.

Reply:
Yes - this is what I wanted to achieve